contracts.json

```diff
@@ -1,13 +1,17 @@
 {
-  "VOTING_TO_CHANGE_KEYS_ADDRESS": "0xc867915e318a41ea8627d9722deb4214a6493445",
-  "VOTING_TO_CHANGE_MIN_THRESHOLD_ADDRESS": "0xbc3b5352e1e02c0f31dbeac47bb4c62969e41d1b",
-  "VOTING_TO_CHANGE_PROXY_ADDRESS": "0x96bcea97bd6cbf1cc6d637796c5c3a7a642bc8fc",
-  "VOTING_TO_MANAGE_EMISSION_FUNDS_ADDRESS": "0x918092ab6f2b6f24b82aeaf797566a222dea4d28",
-  "BALLOTS_STORAGE_ADDRESS": "0xb32fc791d808cb544d9f3229e8333a4627f66c50",
-  "KEYS_MANAGER_ADDRESS": "0xe5a6cdd053d899ff9f2bed1c01595e7fc28fd0f1",
-  "METADATA_ADDRESS": "0xeb35547e1e4630fc6ed9a6f3d4c5e436422f63ea",
-  "PROXY_ADDRESS": "0x9a14adf50abd5a05e274650c5c6da3e93b48180b",
-  "POA_ADDRESS": "0x22e1229a2c5b95a60983b5577f745a603284f535",
-  "EMISSION_FUNDS_ADDRESS": "0xff9c96fdb2aebf16b294775a0723d3b7f1914162",
-  "REWARD_BY_BLOCK_ADDRESS": "0x867305d19606aadba405ce534e303d0e225f9556"
-}
+  "ERC677BridgeTokenRewardable": "0x38A541381D1167b6ACE528eb33adA252ebFBb15E",
+  "ValidatorSetAuRaCode": "0x7B48d8474c9d05a9AF4323c012Cb0E7F4C276673",
+  "ValidatorSetAuRaProxy": "0xB87BE9f7196F2AE084Ca1DE6af5264292976e013",
+  "StakingAuRaCode": "0x365F19752F3deBAF29DCC61D5001AA4f311d700e",
```

Comment:
Just curious, is this property intentionally named `StakingAuRaCode`? I see, that the name of the contract is `StakingAuRa`. Btw, I see mismatches for multiple contracts.

Reply:
That's fine: the lines in this JSON don't define contract names. These are just designations to see addresses of upgradable contracts (`*Code` is an implementation contract address, `*Proxy` is a proxy contract).